internal/cli/atlas/quickstart/prompt.go

```diff
@@ -98,141 +72,70 @@ func newDBUserPasswordQuestion(password, message string) *survey.Question {
 	return &survey.Question{
 		Name: "DBUserPassword",
 		Prompt: &survey.Input{
-			Message: fmt.Sprintf("Database User Password%s:", message),
+			Message: fmt.Sprintf("Database User Password%s", message),
 			Help:    usage.Password,
 			Default: password,
 		},
 		Validate: survey.ComposeValidators(survey.Required, validate.WeakPassword),
 	}
 }
 
-func newSampleDataQuestion(clusterName string) *survey.Confirm {
+func newSampleDataQuestion(clusterName string) survey.Prompt {
```

Comment:
`Prompt` is an interface all `survey` components implement so it makes it easier to change the specific question type if we need to